src/cmd/main.go

```diff
@@ -13,12 +13,20 @@ import (
 	"runtime/pprof"
 	"strings"
 
+	"github.com/VKCOM/noverify/src/cmd/stubs"
 	"github.com/VKCOM/noverify/src/langsrv"
 	"github.com/VKCOM/noverify/src/lintdebug"
 	"github.com/VKCOM/noverify/src/linter"
 	"github.com/VKCOM/noverify/src/meta"
 )
 
+// Line below implies that we have `https://github.com/VKCOM/phpstorm-stubs.git` cloned
+// to the `./src/cmd/stubs/phpstorm-stubs`.
+//
+// Since there is no -include flag, we use -ignore to filter-out all non-php files.
+//
+//go:generate go-bindata -pkg stubs -nometadata -o ./stubs/phpstorm_stubs.go -ignore=.*\.(.|..|[^p][^h][^p]|.{4,})$ ./stubs/phpstorm-stubs/...
```

Comment:
Are you 100% sure there is no better way to do this?

Reply:
Also this regex does not seem to match `meta/.phpstorm.meta.php` file which is really important.

Reply:
>  Are you 100% sure there is no better way to do this?

Which part is implied here?

- go-generate?
- go-bindata?
- or regexp?

I'll try improving regexp. There is no opposite of "ignore" in bindata. The only alternative I see right now is to remove redundant files from the repo or pass results of "find" (or similar utility) as inputs to bindata (although I don't remember whether go generate permits sub-processes).

Reply:
I was talking about go-bindata call itself. Maybe it there is a way to exclude .git

Reply:
Yep, excluding `.git` is trivial. It'll include some redundant files, but I think it's not a problem.
I was aiming to include only php files from start, so the generated file has minimal size.